fmriprep/cli/run.py

```diff
@@ -606,8 +607,9 @@ def build_workflow(opts, retval):
 
     # First check that bids_dir looks like a BIDS folder
     bids_dir = os.path.abspath(opts.bids_dir)
+    layout = BIDSLayout(bids_dir, validate=False)
```

Comment:
Just out of curiosity, why are we using `validate=False`?

Reply:
Assuming that `--skip-bids-validator` was not set. We could actually tail it to that and force going through some validation.

Reply:
ping @chrisfilo 

What about:
```Python
    layout = BIDSLayout(bids_dir, validate=not opts.skip_bids_validator)
```

Reply:
I think `validate=False` is the way to go. `bids-validator` would provide better validation and if it's turned off we should not validate anyway.

Reply:
Oh, okay